components/omega/doc/design/OmegaV1GoverningEqns.md

```diff
@@ -0,0 +1,1103 @@
+**Mass:**
+
+$$
+\frac{\partial {\tilde h}_{i,k} }{\partial t}
++ \nabla \cdot \left(\left[{\tilde h}_k\right]_e {\bf u}_{e,k}\right)
++ \rho_0 \left[ \tilde{W}_{tr} \right]_k
+- \rho_0 \left[ \tilde{W}_{tr} \right]_{k+1}
+= Q_{i,k}.
+$$ (discrete-mass)
+
+In this equation, we have added a mass source term ($Q$) that will include sources like river runoff, sea ice freshwater fluxes, precipitation, and evaporation.
+
+**Tracer:**
+
+$$
+\frac{\partial {\tilde h}_{i,k} \varphi_{i,k}}{\partial t}  & + \nabla \cdot \left(\left[{\tilde h}_{i,k} \varphi_{i,k}\right]_e {\bf u}_{e,k}\right) + \rho_0 \left\{ \left[ \varphi \tilde{W}_{tr} \right]_k - \left[ \varphi \tilde{W}_{tr} \right]_{k+1} \right\} \\
+& = - \nabla \cdot \left({\tilde h}_k \left<\varphi^{\prime} {\bf u}^{\prime}\right>_k \right) - \rho_0 \left\{ \left[\left< \varphi^\prime {\tilde w}_{tr}^{\prime} \right> - \left< \varphi^\prime \tilde{ u}^{\prime}\right> \right]_k - \left[\left< \varphi^\prime {\tilde w}_{tr}^{\prime} \right> - \left< \varphi^\prime \tilde{ u}^{\prime}\right> \right]_{k+1}\right\}.
+$$ (discrete-tracer)
+
+In the tracer equation, we note that surface fluxes (e.g. latent heat fluxes) will appear in the surface value of the vertical turbulent tracer flux (e.g., $\left<\varphi^\prime \tilde{w}_{tr}^\prime\right>_{k=0}$).  The terms on the second line represent the small scale turbulence and the form used in Omega to represent these terms will be discussed in the next section.
+
+**Velocity:**
+
+Omega will only predict the layer average normal velocity, so we drop the bold face on the $u$ terms except for the product of primes, which is specified in the next section.
+
+$$
+\frac{\partial u_{e,k}}{\partial t}
+& + \left[ {\bf k} \cdot \nabla \times u_{e,k} +f_v\right]_e\left(u_{e,k}^{\perp}\right) + \left[\nabla K\right]_e  \\
+& + \frac{\rho_0}{\left[\tilde{h}_{i,k}\right]_e} \left\{ \left[\left(u - u_k\right) \left\{\tilde{W}_{tr} \right\} \right]_{e,k} - \left[  \left(u - u_k\right) \left\{\tilde{W}_{tr} \right\} \right]_{e,k+1} \right\} \\
+& = - \left(\nabla \Phi \right)_{e,k} + \frac{1}{\left[\tilde{h}_k\right]_e} \nabla \left( \tilde{h}_k \alpha_k p_k \right) + \frac{1}{\left[\tilde{h}_k\right]_e} \left\{ \left[ \alpha p \nabla \tilde{z}^{\text{top}}\right]_{e,k} -  \left[ \alpha p \nabla \tilde{z}^{\text{bot}}\right]_{e,k+1} \right\} \\
+&  - \frac{1}{\left[\tilde{h}_{i,k}\right]_e} \nabla \cdot \left( \tilde{h}_k \left< {\bf u}^\prime \otimes {\bf u}^\prime \right>_k \right) - \frac{\rho_0}{\left[\tilde{h}_{i,k}\right]_e}  \left\{ \left[ \left<\mathbf{u}^\prime \tilde{w}_{tr}^\prime \right> - \left< \mathbf{u}^\prime \tilde{ u}^\prime \right> \right]_{e,k} - \left[ \left<\mathbf{u}^\prime \tilde{w}_{tr}^\prime \right> - \left< \mathbf{u}^\prime \tilde{ u}^\prime \right> \right]_{e,k+1} \right\}.
+$$ (discrete-momentum)
+
+**Diagnostic Relations:**
+
+ $$
+ p_{i,k} = p_{i}^{surf} + g\rho_0 \sum_{k'=1}^{k-1} \tilde{h}_{i,k'} + \frac{1}{2} g\rho_0 \tilde{h}_{i,k}
```

Comment:
Is the main reason that we describe p at zmid because we want to use it to calculate layer-averaged alpha?

Reply:
I think this is a question for @sbrus89 and maybe best left for the PGF design doc

Reply:
We will definitely have pressure at interfaces as well in the vertical coordinate class.